PowerEditor/src/ScitillaComponent/UserDefineDialog.h

```diff
@@ -398,20 +398,21 @@ protected :
 class StringDlg : public StaticDialog
 {
 public :
-    StringDlg() : StaticDialog() {};
-    void init(HINSTANCE hInst, HWND parent, const TCHAR *title, const TCHAR *staticName, const TCHAR *text2Set, int txtLen = 0) {
-        Window::init(hInst, parent);
-        _title = title;
-        _static = staticName;
-        _textValue = text2Set;
-        _txtLen = txtLen;
-    };
+    StringDlg() : StaticDialog() {}
+	void init(HINSTANCE hInst, HWND parent, const TCHAR *title, const TCHAR *staticName, const TCHAR *text2Set, int txtLen = 0, bool bGotoCenter = false) {
+		Window::init(hInst, parent);
+		_title = title;
+		_static = staticName;
+		_textValue = text2Set;
+		_txtLen = txtLen;
+		_shouldGotoCenter = bGotoCenter;
+	}
 
     INT_PTR doDialog() {
         return ::DialogBoxParam(_hInst, MAKEINTRESOURCE(IDD_STRING_DLG), _hParent,  dlgProc, reinterpret_cast<LPARAM>(this));
-    };
+    }
 
-    virtual void destroy() {};

```

Comment:
Please keep it. Both ways can be compiled, but for me in header file it is always about declaration.